### ISSUE TITLE
[Feature request]: 

### ISSUE THREAD

**Issue by huntson** [vzhd1701/gridplayer#164]
### Suggestion

Can we have a way of scrubbing all videos in sync from a master playhead rather than controlling each individually?

**Comment by vzhd1701** [comment]
Have you tried this option?

![Clipboard Image (1)](https://github.com/vzhd1701/gridplayer/assets/5212766/62acbfd0-c5cc-4ea2-8590-83715117ce63)

**Comment by huntson** [comment]
Yes.  I found that after I emailed you.  I have, however, been having a consistent issue where, when two clips are playing, specifically they are H.264 encoded, and scrubbing occurs, sooner or later one stops playing all together.  Any thoughts?  It does not appear that my CPU/GPU/HDD are being maxed so I’m not sure where to start.

ian




————————
Ian Liuzzi-Fedun
Liuzzi-Fedun Studios
44 Searing St
Rockville Centre, NY 11570
***@***.***
tel. 516-442-0222
fax. 516-377-8624

**Comment by vzhd1701** [comment]
Looks like a bug, will look into it.

**Comment by huntson** [comment]
I appreciate that.




————————
Ian Liuzzi-Fedun
Liuzzi-Fedun Studios
44 Searing St
Rockville Centre, NY 11570
***@***.***
tel. 516-442-0222
fax. 516-377-8624
